packages/runtime-common/realm-auth-client.ts

```diff
@@ -94,7 +94,7 @@ export class RealmAuthClient {
   }
 
   private async initiateSessionRequest() {
-    return fetch(`${this.realmURL.href}_session`, {
+    return this.loader.fetch(`${this.realmURL.href}_session`, {
```

Comment:
this was necessary in order to perform resolution on the https://cardstack.com/base URL. in general we should always be using the loader's `fetch`.